.github/workflows/main.yml

```diff
@@ -0,0 +1,43 @@
+# Build and run unit tests
+name: CI
+
+on: [ push, pull_request ]
+
+jobs:
+  main:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-18.04, macOS-10.15, windows-2019 ]
+        dc: [ ldc-master, ldc-latest, ldc-1.20.0, dmd-latest, dmd-2.090.1 ]
```

Comment:
> ldc-1.20.0

Is it for workaround?

Reply:
It's just what `crypto` is doing. But crypto doesn't rely on `-preview=in`, and this does, so the `toolChainRequirements` should reflect this

Reply:
I copied it from crypto submodule workflow. 
@Geod24 can we use ldc-1.25.0 here?

Reply:
Perhaps `crypto` should use `ldc-1.25.0` too.